boards/mcb2388/include/board.h

```diff
@@ -0,0 +1,119 @@
+/*
+ * Copyright (C) 2019 Beuth Hochschule für Technik Berlin
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @ingroup     boards_mcb2388
+ * @{
+ *
+ * @file
+ * @brief       Basic definitions for the MCB2388 board
+ *
+ * @author      Benjamin Valentin <benpicco@beuth-hochschule.de>
+ */
+
+#ifndef BOARD_H
+#define BOARD_H
+
+#include "lpc2387.h"
+#include "bitarithm.h"
```

Comment:
This include doesn't seem to be needed here. Am I missing something ?

Reply:
looks like it's save to remove indeed.